frontend/ui/data/koptoptions.lua

```diff
@@ -33,6 +33,7 @@ local KoptOptions = {
                 alternate = false,
                 values = {0, 1, 2},
                 default_value = DKOPTREADER_CONFIG_TRIM_PAGE,
+                enabled_func = function() return require("device"):isTouchDevice() end,
```

Comment:
It's probably better just to include Device at the top (and change Screen to Device.screen)

Then here you should be able to do  `enabled_func = Device.isTouchDevice,`